src/MultiSelect/MultiSelect.svelte

```diff
@@ -394,23 +401,14 @@
           on:keydown
           on:keydown|stopPropagation="{({ key }) => {
             if (key === 'Enter') {
-              if (highlightedIndex > -1) {
-                if (filterable) {
-                  const filteredItemId = filteredItems[highlightedIndex].id;
-                  const filteredItemIndex = sortedItems
-                    .map((item) => item.id)
-                    .indexOf(filteredItemId);
-
-                  sortedItems = sortedItems.map((item, i) => {
-                    if (i !== filteredItemIndex) return item;
-                    return { ...item, checked: !item.checked };
-                  });
-                } else {
-                  sortedItems = sortedItems.map((item, i) => {
-                    if (i !== highlightedIndex) return item;
-                    return { ...item, checked: !item.checked };
-                  });
-                }
+              if (highlightedId) {
+                const filteredItemIndex = sortedItems.findIndex(
```

Comment:
findIndex isn't supported by IE iirc. do you expect people to bring polyfills or should this be rewritten to be more compatible?

Reply:
We also use `find`. I think it's acceptable to focus on modern browsers. Also, MS will no longer support IE after June 15, 2022.